lib/proxy_util.js

```diff
@@ -55,20 +55,20 @@ exports.getCompareAndLogEnvAndAgentProxies = function (agentOptions) {
       ' protocol=' + agentOptions.protocol + ' proxy=' + proxyHostAndPort
       : ' proxy=' + proxyHostAndPort;
     const proxyUsername = agentOptions.user ? ' user=' + agentOptions.user : '';
+    const proxyString = `${Util.exists(proxyUsername) ? `${agentOptions.user}:${agentOptions.password}@` : ''}${proxyHostAndPort}`.toLowerCase();
```

Comment:
using `proxyHost: 192.168.100.1` and `proxyPort: 80` this ends up being `undefined:undefined@192.168.100.1:80` and i'm unsure if it's expected or not

Reply:
I fixed it. Thank you for letting me know.